samples/security/vulnerability findings/jira_tickets_for_critical_vulnerabilities.yaml

```diff
@@ -0,0 +1,163 @@
+metadata:
+  version: "1"
+  dependencies:
+    apps:
+      - id: dynatrace.automations
+        version: ^1.728.0
```

Comment:
version is not released, but at least everything seems to work after importing the WF in prod

Reply:
ah - is released - just not deployed in demo.live - all good